source/mir/ndslice/selection.d

```diff
@@ -1713,9 +1713,12 @@ template IotaSlice(size_t N)
 
 // undocumented
 // zero cost variant of `std.range.iota`
+///
 struct IotaMap()
```

Comment:
Dscanner rightfully complains about that as you shouldn't have anything that is public and undocumented...